src/README.md

```diff
@@ -0,0 +1,42 @@
+
+# S2 Geometry Library
```

Comment:
This seems like a shortened version of https://github.com/google/s2geometry/blob/master/README.md
better to combine with that

Reply:
7.0.0 for bzlmod